KsApi/extensions/ApolloClient+RAC.swift

```diff
@@ -15,6 +16,9 @@ extension ApolloClient {
       self.fetch(query: query, cachePolicy: .fetchIgnoringCacheCompletely) { result in
         switch result {
         case let .success(response):
+          if let error = response.errors?.first?.errorDescription {
```

Comment:
Successful responses from GraphQL can contain validation errors, so we handle that here.